cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py

```diff
@@ -45,6 +49,25 @@ def __append_input_tags(self, current_tags: list = None):
                     input_tags.append(current_item)
         return input_tags
 
+    def __add_cluster_id_tag(self, tags: list):
+        """
+        This method add cluster_id tag if not present
+        :param tags:
+        :type tags:
+        :return:
+        :rtype:
+        """
+        cluster_id_tag = get_tag_value_from_tags(tags=tags, tag_name=self.CLUSTER_ID_COST_ALLOCATION_TAG)
+        if not cluster_id_tag:
+            cluster_name = ''
+            for tag in tags:
+                if self.cluster_prefix in tag.get('Key'):
+                    cluster_name = tag['Key']
+                    break
+            if cluster_name:
+                cluster_name = cluster_name.split('/')[-1]
+                tags.append({'Key': self.CLUSTER_ID_COST_ALLOCATION_TAG, 'Value': cluster_name})
```

Comment:
Are you sure the cluster_name is the cluster_id ?

Reply:
<img width="343" alt="Screenshot 2024-05-22 at 4 38 16 PM" src="https://github.com/redhat-performance/cloud-governance/assets/97395760/dff38d82-32e4-4b1b-9fce-d089d82904c2">

Reply:
cluster_name == cluster_id ?

Reply:
Pls add comment cluster_name == cluster_id